src/components/grid-aware/ImageHeader/ImageHeader.module.css

```diff
@@ -0,0 +1,139 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
+  }
+}
+
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+}
+
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding-top: 30px;
+  }
+}
+
+
+/* Main grid areas */
+
+.gridAreaText {
+  grid-column: 2 / span 6;
+  grid-row: 1;
+}
+
+.gridAreaImage1 {
+  grid-column: 9 / span 4;
+  grid-row: 1;
+}
+
+.gridAreaImage2 {
+  grid-column: 10 / span 3;
+  grid-row: 1;
+}
+
+@media (--tablet-and-down) {
+  .gridAreaImage2 {
+    text-align: right;
+  }
+}
+
+
+/* Left grid area */
+
+.title {
```

Comment:
This is a little bit subtle, but you'll also need to zero out the bottom margin on the `.title`, since `<h1>`s come with some default bottom margin. This is only visually noticeable on the mobile view, where the default 20px bottom margin of the `<h1>` is larger than the 15px top margin from the `.subtitle`, which means that 20px bottom margin wins and sets the total spacing between the elements.